src/main/resources/application.conf

```diff
@@ -9,7 +9,7 @@ mantis {
   datadir = ${user.home}"/.mantis"
 
   # The unencrypted private key of this node
-  node-key-file = ${mantis.datadir}"/nodeId.keys"
+  node-key-file = ${mantis.datadir}"/node.key"
```

Comment:
it may be worth to update this in .gitignore as well

Reply:
I don't see why we'd need that file it `.gitignore`. Must be a very old entry. Removed